src/aat/java/uk/gov/hmcts/reform/coh/idam/IdamClient.java

```diff
@@ -2,9 +2,13 @@
 
 public interface IdamClient {
 
-    void createAccount(String email, String role);
+    void createAccount(String email, String role, String password);
```

Comment:
Do like this -> void createAccount(String email, String password, String role);
More logical to have the password next to the username

Reply:
Done.